### ISSUE TITLE
How to detect the speed of the model?

### ISSUE THREAD

**Issue by liguilan1227** [pierluigiferrari/ssd_keras#360]
I tried this code and got a high accuracy by evaluation.Meanwhile, How could I detect the speed of the model, I mean the object about FPS eval. Could you help me please?

**Comment by hiren-2911** [comment]
Hey! is the model working for you?

**Comment by hiren-2911** [comment]
Can you please help me i am getting some errors which i have mentioned in the issues

**Comment by liguilan1227** [comment]
> Can you please help me i am getting some errors which i have mentioned in the issues
I would like to help you but I didnot meet this error as you discribe...

**Comment by hiren-2911** [comment]
Did you train the model on your own custom data or are you just using pre-trained ?
if yes, can you please share you code with me?
It would be of great help.
I am using the latest version of this model that is the ssd-keras-0.9.0.

**Comment by vk1996** [comment]
> I tried this code and got a high accuracy by evaluation.Meanwhile, How could I detect the speed of the model, I mean the object about FPS eval. Could you help me please?

Try this
```
start_time=time.time()
def calc_fps():
    elapsed_time=time.time()-start_time
    return frame_counter//elapsed_time
frame_counter=0
while True:
         model.predict(input_array)
         frame_counter+=1
         print(calc_fps())

```
